### ISSUE TITLE
[Improvement] Support ACF's get_field_object by TimberPost

### ISSUE THREAD

**Issue by piotrkulpinski** [timber/timber#1597]
Hi,

Would be great if Timber would support get_field_object the same way it support get_field function: 
https://www.advancedcustomfields.com/resources/get_field_object/

What I'd love to have would be something like that:
```
{% set field = post.get_field_object('field') %}

{{ field.label }}
{{ field.value }}
```

Let me know what you think.

**Comment by palmiak** [comment]
https://github.com/palmiak/timber/tree/get_field_object

Try this - it should be working just as you want to:
```php
{% set fields = post.get_field_object('field_name') %}
{{ function( 'var_dump', fields ) }}
```

**Comment by piotrkulpinski** [comment]
@palmiak Thanks!

Maybe you could create a pull request to include that in the core?

cc @jarednova

**Comment by gchtr** [comment]
This feature was released in [1.6.0](https://github.com/timber/timber/releases/tag/1.6.0).
